### PR TITLE
Feature/game page

### PR DESCRIPTION
**Pull request by alextfriedman**
Score counter & game over page updates merging to main

### CODE REVIEW

**Review by mwazir — APPROVED** [review]
Looks good!

File: src/Components/Game.js [hunk 1]
```diff
@@ -105,7 +108,8 @@ const Game = (props) => {
 
   return (
     <div>
-      { isGameOver ? <GameOverDisplay /> : 
+      <Score score={score} setScore={setScore} />
+      { isGameOver ? <GameOverDisplay scoreSubmitted={scoreSubmitted} setScoreSubmitted={setScoreSubmitted}/> :
```
Comment (itsninam) [inline src/Components/Game.js:112]: Looks great!

File: src/Components/Game.js [hunk 2]
```diff
@@ -2,11 +2,12 @@ import { useState, useEffect } from "react";
 import axios from "axios";
 import GameOverDisplay from "./GameOverDisplay";
 import QuestionsDisplay from "./QuestionsDisplay";
+import Score from "./Score";
 
 const Game = (props) => {
   //Examples of Homophone Pairs https://www.englishclub.com/pronunciation/homophones-list.htm
   
-  const { initialWords } = props;
+  const { initialWords} = props;
```
Comment (benbourgon) [inline src/Components/Game.js]: Looks like we lost a space on the right side of initialWords

File: src/Components/Game.js [hunk 3]
```diff
@@ -105,7 +108,8 @@ const Game = (props) => {
 
   return (
     <div>
-      { isGameOver ? <GameOverDisplay /> : 
+      <Score score={score} setScore={setScore} />
```
Comment (benbourgon) [inline src/Components/Game.js:111]: C😎😎l conditional rendering for the game over/submitted components

**Review by benbourgon — APPROVED** [review]

File: src/Components/Game.js (hunk 2 above)
Comment (alextfriedman) [reply src/Components/Game.js]: got it!

**Pull request merged**